homeassistant/helpers/hassio.py

```diff
@@ -0,0 +1,11 @@
+"""Supervisor helper."""
+
+import os
+
+from homeassistant.core import callback
+
+
+@callback
+def is_hassio() -> bool:
+    """Return true if on Hass.io."""
+    return "HASSIO" in os.environ
```

Comment:
```suggestion
    return "SUPERVISOR" in os.environ
```

Reply:
Lol oh right, that breaks all the tests. Gimme a sec, will fix